pkg/edvprovider/couchdbedvprovider/couchdbedvprovider.go

```diff
@@ -21,6 +22,8 @@ import (
 
 const mapDocumentIndexedField = "IndexName"
 
+var logger = log.New("EDV-CouchDBProvider")
```

Comment:
as per convention we followed in other projects, we use lowercase in module names.